enterprise/utils.py

```diff
@@ -365,39 +369,6 @@ def get_catalog_admin_url_template(mode='change'):
     return None
 
 
-def build_notification_message(template_context, template_configuration=None):
```

Comment:
this function boils down to one line, so I removed it.. and we just now call the template_config method in the calling site